### PR TITLE
[12] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by ainobs**
Screenshot (34)
Screenshot (35)
Screen Shot 2020-09-01 at 19 26 19
Screen Shot 2020-09-01 at 19 26 32
Screen Shot 2020-09-01 at 20 15 47
Screen Shot 2020-09-01 at 20 14 22

## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/peterIyiola/css/contact.css [hunk 1]
```diff
@@ -214,5 +214,4 @@ footer    {
 
 
 }
-    
-
+
```
Comment (adaugochi) [inline submissions/peterIyiola/css/contact.css:217]: You ought not to tamper with someone else file. So, revert it back to the way it was. Add the lines you removed.

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by ToluAkin** [comment]
@ainobs On small screen, the submit button is on another block while there is still enough space for it to fit.Adjust this and reupload another screenshot.

File: submissions/victorAienobe/css/contact.css [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+#text2 {
+    float: left;
+    margin: 20px 0 0 37px;
+}
+.email {
+    margin: 20px auto;
+}
+.text {
+    text-align: center;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```
Comment (ToluAkin) [inline submissions/victorAienobe/css/contact.css:64]: @ainobs Take this to the body tag so that all the elements on the page should use that font.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/peterIyiola/css/contact.css (hunk 1 above)
Comment (ToluAkin) [reply submissions/peterIyiola/css/contact.css:217]: @ainobs You need to fix this

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**